extensions/mssql/src/ui/dialogBase.ts

```diff
@@ -147,6 +147,17 @@ export abstract class DialogBase<DialogResult> {
 		return this.createInputBox(ariaLabel, textChangeHandler, value, enabled, 'password', width);
 	}
 
+	protected createInputBoxWithProperties(ariaLabel: string, textChangeHandler: (newValue: string) => Promise<void>, properties: azdata.InputBoxProperties, customValidation?: () => Promise<boolean>): azdata.InputBoxComponent {
```

Comment:
We should not have two, are you going to follow up with removing createInputBox and using this one instead?

Reply:
yes, I'll remove the other one. I created an issue tracking this: https://github.com/microsoft/azuredatastudio/issues/23742